templates/ehub.json

```diff
@@ -66,6 +66,20 @@
                 "description": "Please use the actual name if using existing event hub."
             }
         },
+        "Event Hub Retention": {
+            "type": "Int",
+            "defaultValue": 7,
+            "metadata": {
+                "description": "Only applies if new event hub is created! Data retention time in days between 1 and 7."
+            }
+        },
+        "Event Hub Partition Count": {
+            "type": "Int",
+            "defaultValue": 32,
+            "metadata": {
+                "description": "Only applies if new event hub is created! Event hub partitions count between 2 and 32. Recommentadion: balance 1:1 throughput units and partitions to achieve optimal scale."
```

Comment:
Perhaps use a similar format to earlier descriptions; something like:
"Optional. Event hub partitions count, between 2 and 32. Leave empty if reusing an existing Event Hub. Recommendation: balance 1:1 throughput units and partitions to achieve optimal scale."

Reply:
Anyway, spelling: should be 'recommendation'